docs-new/docs/graphql/core/actions/action-permissions.mdx

```diff
@@ -0,0 +1,82 @@
+---
+sidebar_label: Actions permissions
+sidebar_position: 7
+description: Permissions for Hasura actions
+keywords:
+  - hasura
+  - docs
+  - actions
+  - permissions
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import Thumbnail from "@site/src/components/Thumbnail/Thumbnail";
+
+# Actions permissions
+
+## Introduction
+
+As with the other fields in the GraphQL schema, users need to be given
+access to an action.
+
+## Set action permissions
+
+<Tabs className="api-tabs">
+<TabItem value="console" label="Console">
+
+Head to the `Actions -> [action-name] -> Permissions` tab in the
+console.
+
+<Thumbnail
+  src="/img/graphql/core/actions/actions-permissions.png"
+  alt="Console action permission"
+/>
+
+Hit `Save` to give the role permission to access the action.
+</TabItem>
+<TabItem value="cli" label="CLI">
+
+Go to `metadata/actions.yaml` in the Hasura project directory.
+
+Update the definition of the `insertAuthor` action as:
+
+```yaml {6-8}
+- actions
+  - name: insertAuthor
+    definition:
+      kind: synchronous
+      handler: '{{ACTIONS_BASE_URL}}/insertAuthor'
+    permissions:
+    - role: user
+    - role: publisher
+```
+
+Save the changes and run `hasura metadata apply` to set the permissions.
+
+</TabItem>
+<TabItem value="api" label="API">
+
+Action permissions can be set by using the [metadata_create_action_permission](/graphql/core/api-reference/metadata-api/actions.mdx) metadata API:
```

Comment:
link label mismatch

Reply:
and the link is to a subsection. But the link here is to a file only

Please add #hash-param to link to respective sub section

Reply:
Label - `create_action_permission`